test/10.js

```diff
@@ -0,0 +1,10 @@
+const expect = require('chai').expect;
+let solution = require('../solutions/10').solution;
+//solution = require('../yourSolution').solution;
+
+describe('Test 1: Given [1,2,3,9,8,7]', () => {
```

Comment:
Add more test case. Particularly edge cases that might break your app. Always test for edge cases.

I would add these: `[1], [3], [1,3]`